### PR TITLE
16.11 External Packages Insertion

### PR DESCRIPTION
**Pull request by Haplois**
## Description
Updated VS Externals

### CODE REVIEW

**Review by cvpoienaru — APPROVED** [review]
Tentatively approving this but please check if all the localization changes should've been included as well before merging.

**Comment by Haplois** [comment]
They are just formatting changes - I checked. This way it'll be easier on us if we need to service it again.

**Pull request merged**
